tests.robot

```diff
@@ -0,0 +1,15 @@
+*** Settings ***
+# Don't change these settings when trying out Qentinel Pace for the first time
+Documentation       Test suites for Pace starter.
+Library             QWeb
+Suite Setup         Open Browser       about:blank       chrome
+Suite Teardown      Close All Browsers
+
+
+*** Test Cases ***
+# You can add your own test cases here
+Test the global home page
+    [Documentation]     Accepts the cookie policy and verifies home page text.    # Explain the test
+    GoTo                https://qentinel.com                                      # You can change this URL to test any other website
+    ClickText           Accept                                                    # Click any visible text on the page
+    VerifyText          Automate Software Testing with Qentinel Pace              # Verify that the page contains the given text
```

Comment:
It seems that I'm getting the Finnish version of the page.  The previous version had `ClickText  Change location` and `ClickText  Global` to fix the language of the page.  To shorten the test, I wonder if we could use some phrase that appears regardless of the language.  `VerifyText  Code confidence by Qentinel`?

Reply:
Setting `GoTo   https://qentinel.com/?noredirect=true` would stick to the English website, else it redirects based on your browser's default language (if I'm not mistaken).

Reply:
That's interesting. For me the test works and I still get the global site, even though I set my browser's language to Finnish. In live-testing I clicked open the robot's browser's settings and the language was set to English, as expected.
If I'm correct, the robot is running in Ireland (according to my Google Analytics) so shouldn't the language always be English by default?
If the problem persists, I can change the link to [https://qentinel.com/?noredirect=true](https://qentinel.com/?noredirect=true), but it might be slightly more confusing.

Reply:
I still think we should also remove Suite Setup and Suite Teardown if we remove Appstate. They are logically the same and will be explained at the same time in trainings etc.

Test case can just use OpenBrowser with the end url directly, no need to open about:blank first and then goto the real page.

Reply:
> If the problem persists, I can change the link to https://qentinel.com/?noredirect=true, but it might be slightly more confusing.

I propose that we use a mock website instead. I can set on up on github pages.

Reply:
So the test would go to https://qentinelqi.github.io/ and test stuff there, like `VerifyText    A website to test Qentinel Pace.`?
I think that's a pretty cool idea. Then the Starter suite won't be dependent on Qentinel's site's content either, in case it changes.
The content of the mock website seems like a generated default though and it doesn't have much to test currently, so it should probably be improved first,
It could technically even have some random examples where you can test more practical interactive things, like _After clicking this button, is the result 1?_ That might be impossible with markdown though and this was just a random idea that I thought of.

Reply:
> I still think we should also remove Suite Setup and Suite Teardown if we remove Appstate. They are logically the same and will be explained at the same time in trainings etc.

If you think so, then sure. I still think that this is slightly longer (if the user adds a second or more test cases) and requires more from the user than the `SuiteSetup` and `SuiteTeardown`, as those do it automatically  _for_ the user. Opening and closing the browser also aren't really related to the test case itself (in my opinion), instead, they make preparations before the test case. These are just my personal opinions though.

Anyway, I'm guessing that this is your proposed structure?
```robotframework
*** Settings ***
# Don't change these settings when trying out Qentinel Pace for the first time
Documentation       Test suites for Pace starter.
Library             QWeb


*** Test Cases ***
# You can add your own test cases here
Test the global home page
    [Documentation]     Accepts the cookie policy and verifies home page text.    # Explain the test
    OpenBrowser         https://some-url    chrome                                # Open browser in a given URL and browser
    ClickText           Accept                                                    # Click any visible text on the page
    VerifyText          Automate Software Testing with Qentinel Pace              # Verify that the page contains the given text 
    CloseAllBrowsers                                                              # Close browsers
```
Also, should the `Documentation` in `***Settings***` be kept or removed? Originally I was confused, as I thought it was documentation for the settings and not the entire test file (I'm still not sure). I also think it's a bit unnecessary.